website/templates/emails/new_public_project.html.mako

```diff
@@ -8,7 +8,7 @@
         Congratulations on making a public project on the Open Science Framework (OSF)! Now that your project “${project_title}" is public, you’ll be able to take advantage of more OSF features:
         <ul>
             <li><a href="${osf_url}${nid}/files/">The number of downloads of your files will be automatically logged for you</a></li>
-            <li><a href="${osf_url}${nid}/settings/">You can enable other OSF users to comment on your work</a></li>
+            <li><a href="${osf_url}${nid}/settings/">You can enable other OSF users to comment on your work or update your own comment notification settings</a></li>
```

Comment:
The default setting for commenting is now that other users can comment on your project, so I actually think we should remove this bullet and replace it with: 

<li><a href="${osf_url}${nid}/analytics/">You can see analytics on how many others are accessing your projects</a></li>

I know this is out of scope/not included in the spec, so if you'd like me to put in a separate ticket I'm happy to do so.